frontends/mit-learn/src/pages/HomePage/HeroSearch.tsx

```diff
@@ -194,40 +202,37 @@ const HeroSearch: React.FC = () => {
             onSubmit={onSearchSubmit}
           />
           <LinksContainer>
-            <TrenderingContainer>
-              <Typography
-                sx={{ marginRight: "8px" }}
-                typography={{ xs: "subtitle4", md: "subtitle3" }}
-              >
-                Trending
-              </Typography>
-              {SEARCH_CHIPS.map((chip) => (
-                <SquaredChip
-                  noBorder
-                  key={chip.label}
-                  variant={chip.variant}
-                  size="medium"
-                  label={chip.label}
-                  href={chip.href}
-                />
-              ))}
-            </TrenderingContainer>
-            <BrowseContainer>
-              <SquaredChip
-                grow
-                variant="outlined"
-                size="medium"
-                label="Browse by Topic"
-                href="/topics/"
-              />
-              <SquaredChip
-                grow
-                variant="filled"
-                size="medium"
-                label="Explore All"
-                href="/search/"
-              />
-            </BrowseContainer>
+            <BrowseByTopicContainer>
+              <BrowseByTopicText>
+                or browse by{" "}
+                <TopicLink href="/topics/" color="red">
+                  Topic
+                </TopicLink>
+              </BrowseByTopicText>
+            </BrowseByTopicContainer>
+            <TrendingContainer>
+              {SEARCH_CHIPS.map((chip) =>
+                chip.variant === "outlinedWhite" ? (
```

Comment:
If you want to skip the ternary, this you could make `StyledChip`

```
/**
 * Apply some overrides to `outlinedWhite`.
 */
const StyledChipLink = styled(ChipLink)(({ theme, variant }) => [
  variant === "outlinedWhite" && {
    borderColor: theme.custom.colors.lightGray2,
    color: theme.custom.colors.silverGrayDark,
  },
])
```

@steven-hatch Should we consider updating the [`outlinedWhite` variant ](https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=3007-43275&m=dev)we use elsewhere? (Or perhaps use that variant w/o customization here.)